modules/sdk-api/src/bitgoAPI.ts

```diff
@@ -1524,12 +1539,15 @@ export class BitGoAPI implements BitGoBase {
   /**
    * Gets the user's private keychain, used for receiving shares
    */
-  async getECDHSharingKeychain(): Promise<any> {
-    const result = await this.get(this.url('/user/settings')).result();
-    if (!result.settings.ecdhKeychain) {
-      return new Error('ecdh keychain not found for user');
+  async getECDHKeychain(ecdhKeychainPub?: string): Promise<any> {
```

Comment:
Update the comment for this fn. It is no longer only for receiving shares. it is also for signing now.